scaletest/templates/scaletest-runner/scripts/prepare.sh

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+set -euo pipefail
+
+[[ $VERBOSE == 1 ]] && set -x
+
+echo "Preparing scaletest workspace environment..."
+touch /tmp/.scaletest_preparing
```

Comment:
nit: Should we move `.scaletest_*` files under the SCALETEST_RUN_DIR or `/tmp/scaletest`? Then you wouldn't need to treat them as hidden files (might be easier to `ls` these flags).